ArabicScript/kurdish.yaml

```diff
@@ -0,0 +1,108 @@
+name: Kurdish
+languages: ckb
+rows:
+  - letters: # rowkeys_kurdish1.xml
+    # U+0642: ق ARABIC LETTER QAF
+    - {type: case, normal: ["\u0642", "`"], shiftedManually: ["`", "\u0642"]}
+
+    # U+0648: و ARABIC LETTER WAW
+    - {type: case, normal: ["\u0648", "\u0648\u0648"], shiftedManually: ["\u0648\u0648", "\u0648"]}
+
+    # U+06D5: ە ARABIC LETTER AE
+    # U+064A: ي ARABIC LETTER YEH
+    - {type: case, normal: ["\u06d5", "\u064a"], shiftedManually: ["\u064a", "\u06d5"]}
+
+    # U+0631: ر ARABIC LETTER REH
+    # U+0695: ڕ ARABIC LETTER REH WITH SMALL V BELOW
+    - {type: case, normal: ["\u0631", "\u0695"], shiftedManually: ["\u0695", "\u0631"]}
+
+    # U+062A: ت ARABIC LETTER TEH
+    # U+0637: ط ARABIC LETTER TAH
+    - {type: case, normal: ["\u062a", "\u0637"], shiftedManually: ["\u0637", "\u062a"]}
+
+    # U+06CC: ی ARABIC LETTER FARSI YEH
+    # U+06CE: ێ ARABIC LETTER YEH WITH SMALL V
+    - {type: case, normal: ["\u06cc", "\u06ce"], shiftedManually: ["\u06ce", "\u06cc"]}
+
+    # U+0626: ئ ARABIC LETTER YEH WITH HAMZA ABOVE
+    # U+0621: ء ARABIC LETTER HAMZA
+    - {type: case, normal: ["\u0626", "\u0621"], shiftedManually: ["\u0621", "\u0626"]}
+
+    # U+062D: ح ARABIC LETTER HAH
+    # U+0639: ع ARABIC LETTER AIN
+    - {type: case, normal: ["\u062d", "\u0639"], shiftedManually: ["\u0639", "\u062d"]}
+
+    # U+06C6: ۆ ARABIC LETTER OE
+    # U+0624: ؤ ARABIC LETTER WAW WITH HAMZA ABOVE
+    - {type: case, normal: ["\u06c6", "\u0624"], shiftedManually: ["\u0624", "\u06c6"]}
+
+    # U+067E: پ ARABIC LETTER PEH
+    # U+062B: ث ARABIC LETTER THEH
+    - {type: case, normal: ["\u067e", "\u062b"], shiftedManually: ["\u062b", "\u067e"]}
+
+  - letters: # rowkeys_kurdish2.xml
+    # U+0627: ا ARABIC LETTER ALEF
+    # U+0622: آ ARABIC LETTER ALEF WITH MADDA ABOVE
+    - {type: case, normal: ["\u0627", "\u0622"], shiftedManually: ["\u0622", "\u0627"]}
+
+    # U+0633: س ARABIC LETTER SEEN
+    # U+0634: ش ARABIC LETTER SHEEN
+    - {type: case, normal: ["\u0633", "\u0634"], shiftedManually: ["\u0634", "\u0633"]}
+
+    # U+062F: د ARABIC LETTER DAL
+    # U+0630: ذ ARABIC LETTER THAL
+    - {type: case, normal: ["\u062f", "\u0630"], shiftedManually: ["\u0630", "\u062f"]}
+
+    # U+0641: ف ARABIC LETTER FEH
+    # U+0625: إ ARABIC LETTER ALEF WITH HAMZA BELOW
+    - {type: case, normal: ["\u0641", "\u0625"], shiftedManually: ["\u0625", "\u0641"]}
+
+    # U+06AF: گ ARABIC LETTER GAF
+    # U+063A: غ ARABIC LETTER GHAIN
+    - {type: case, normal: ["\u06af", "\u063a"], shiftedManually: ["\u063a", "\u06af"]}
+
+    # U+0647: ه ARABIC LETTER HEH
+    # U+06BE: ھ ARABIC LETTER HEH DOACHASHMEE
+    # U+200C: ‌ ZERO WIDTH NON-JOINER
+    - {type: case, normal: ["\u06be|\u0647", "!icon/zwnj_key|\u200c"], shiftedManually: ["!icon/zwnj_key|\u200c", "\u0647"]}
```

Comment:
Can you explain the change here? (the added 06BE, originally this was `{type: case, normal: ["\u0647", "!icon/zwnj_key|\u200c"], shifted: ["!icon/zwnj_key|\u200c", "\u0647"]}`)

Reply:
In the Gboard layouts for Kurdish (and in Noto fonts), the isolated form of U+0647 HEH ه looks like U+06BE HEH DOACHASHMEE ھ, to differentiate it from U+06D5 AE ە. Otherwise, both HEH and AE look the same in isolated form (on the key labels), so it's better to follow Gboard and make the key label look like HEH DOACHASHMEE.

Current Central Kurdish layout (notice the similarity of top row key 3 and row 2 key 6, counting from the left side):
![image](https://github.com/user-attachments/assets/57475320-79e5-4a52-a596-04815e2d4f4d)

Gboard Kurdish (Iraq) layout (HEH (top row key 8) is drawn differently from AE (bottom row key 6)):
![image](https://github.com/user-attachments/assets/5d09a3a0-72ea-449a-8890-56b08b398e25)

Reply:
Interesting, thank you